test/unit/views/PdfLinksNav.test.js

```diff
@@ -85,27 +85,22 @@ describe('User language: English. Component <PdfLinksNav>: two ladok round ids,
 
   test('renders 2 memo pdf links and one course analysis', () => {
     const twoDocLinks = getAllByRole('link')
-    expect(twoDocLinks.length).toBe(3)
+    expect(twoDocLinks.length).toBe(2)
```

Comment:
why amount of links decreased?

Reply:
fixed this issue, it was issue with the paramiters of render. Now it is reverted back to original length.